salt/metalk8s/addons/dex/deployed/chart.sls

```diff
@@ -29,6 +31,31 @@ stringData:
       https: 0.0.0.0:5556
       tlsCert: /etc/dex/tls/https/server/tls.crt
       tlsKey: /etc/dex/tls/https/server/tls.key
+    connectors:
+    - config:
+        bindDN: {% endraw -%}{{ dex.spec.connectors.config.bindDN }}{%- raw %}
+        bindPW: {% endraw -%}{{ dex.spec.connectors.config.bindPW }}{%- raw %}
+        groupSearch:
+          baseDN: {% endraw -%}{{ dex.spec.connectors.config.groupSearch.baseDN }}{%- raw %}
+          filter: (objectClass=group)
+          nameAttr: {% endraw -%}{{ dex.spec.connectors.config.groupSearch.nameAttr }}{%- raw %}
+          userMatchers:
+            groupAttr: {% endraw -%}{{ dex.spec.connectors.config.groupSearch.userMatchers.groupAttr }}{%- raw %}
+            userAttr: {% endraw -%}{{ dex.spec.connectors.config.groupSearch.userMatchers.userAttr }}{%- raw %}
+        host: {% endraw -%}{{ dex.spec.connectors.config.host }}{%- raw %}
+        rootCAData: {% endraw -%}{{ dex.spec.connectors.config.rootCAData }}{%- raw %}
+        startTLS: true
+        userSearch:
+          baseDN: {% endraw -%}{{ dex.spec.connectors.config.userSearch.baseDN }}{%- raw %}
+          emailAttr: {% endraw -%}{{ dex.spec.connectors.config.userSearch.emailAttr }}{%- raw %}
+          filter: (&(objectclass=user))
+          idAttr: {% endraw -%}{{ dex.spec.connectors.config.userSearch.idAttr }}{%- raw %}
+          nameAttr: {% endraw -%}{{ dex.spec.connectors.config.userSearch.nameAttr }}{%- raw %}
+          username: {% endraw -%}{{ dex.spec.connectors.config.userSearch.username }}{%- raw %}
+        usernamePrompt: SSO Username
```

Comment:
Wouldn't it make sense for this to be configurable? What if the login system is not known as `SSO` within the organization?

Reply:
Yes, going to make this configurable